client/src/databrowser/dictionary.parse.js

```diff
@@ -114,7 +111,15 @@ export function parseDictionary(input) {
 				}
 
 				if (type == 'integer' || type == 'float') {
-					for (const k in terms[termId].values) terms[termId].values[k].uncomputable = true
+					// term is numeric. keys declared in .values{} should be uncomputable categories
+					for (const k in terms[termId].values) {
+						const tmp = Number(k)
+						if (Number.isNaN(tmp)) {
+							// this is by design so that all values in the db to match the column type for that value, otherwise the generated SQL statements would always have to include
+							throw `Uncomputable category of a numeric term is required to be a number (here uses non-numeric value of ${k}).`
```

Comment:
@siosonel somehow this does not throw when i tested with such data. anything wrong?

Reply:
@xzhou82 I played around with different data and it caught everything except ''. Changing the line to `Number.isNaN(tmp) || tmp == ''` fixed the issue.

Reply:
thanks this is now addressed in new commit